examples/claude-try-it/install.ps1

```
@@ -373,6 +451,360 @@ function Find-FreePort {
+    } catch {
+        return $false
+    } finally {
+        $env:PGPASSWORD = $null
+        $ErrorActionPreference = $prevPref
+    }
+}
+
+# --- List user databases -------------------------------------------------
+
+function Get-UserDatabases {
+    param([int]$Port, [string]$User, [string]$Password)
```

PSScriptAnalyzer (1.25.0)

[error] Function 'Get-UserDatabases' has both Username and Password parameters. Either set the type of the Password parameter to SecureString or replace the Username and Password parameters with a Credential parameter of type PSCredential. If using a Credential parameter in PowerShell 4.0 or earlier, please define a credential transformation attribute after the PSCredential type attribute.

(PSAvoidUsingUsernameAndPasswordParams)